### DISCUSSION TITLE
Redis NOAUTH issue

### DISCUSSION THREAD

**Discussion by exceptrohit** [redis/lettuce#2466] [General]
I am getting this error periodically in my service :
io.lettuce.core.RedisCommandExecutionException: NOAUTH Authentication required ,trace : java.util.concurrent.ExecutionException: io.lettuce.core.RedisCommandExecutionException: NOAUTH Authentication required. 

This error gets mitigated after restarting the service and the credentials provided are correct. What is the reason for this and the fix?

**Comment by tishun** [comment]
The provided information is not enough to determine what the issue is.
Is the message shown when the client is attempting a reconnect? 

It is likely that the credentials used when reconnecting are stale / incorrect and when the service is restarted they get updated.
